src/inclusion.md

```diff
@@ -52,6 +52,13 @@ PyCon UK will be a friendly environment for parents and babies.
   </p>
 </div>
 
+<div class="box box_yellow">
+  <h3>Older Children</h3>
```

Comment:
This might be better titled as "Under 18s" as someone who is 17 may well not like to consider themself in the "Older children" bucket and thus might not mentally note this section as applying to them.